tutorials/kubernetes/install-kubernetes-with-conjure-up.md

```diff
@@ -0,0 +1,159 @@
+
+```bash
+$ conjure-up
+```
+
+You will be presented with a list of recommended spells. From this list, select "The Canonical Distribution of Kubernetes" and press enter.
+
+![IMAGE](https://assets.ubuntu.com/v1/042c1dd7-select-canonical-distribution-of-kubernetes.png)
+
+## Select your cloud
+Duration: 3
+
+conjure-up will then ask where you would like to deploy your applications.
+
+For this tutorial we're going to demonstrate installation on a public cloud using Amazon AWS.
+
+There will be minor differences between the steps for the various cloud providers but generally consists of steps such as choosing a region and entering the credentials for the cloud provider.
+
+![IMAGE](https://assets.ubuntu.com/v1/43eb777a-Screenshot+from+2017-08-18+11-57-29.png)
+
+
+## Deploy the controller and applications
+Duration: 18
+
+Once you have configured the cloud provider that you are using you will then be presented with a screen titled "CHOOSE A CONTROLLER OR CREATE NEW".
+
+For the purposes of this tutorial we'll start from scratch and select "Deploy New Self-Hosted Controller".
+
+![IMAGE](https://assets.ubuntu.com/v1/ed7970a6-new-controller.png)
+
+You will be presented with a list of the applications required for a Kubernetes installation. You are presented with an option to individually configure the options but we can simply select "Deploy all remaining applictaions" and use the suggested default configuration.
+
+![IMAGE](https://assets.ubuntu.com/v1/73946c3f-deploy-all.png)
+
+You will see verbose output of the machines being automatically provisioned and the Juju controller will install the requisite charm bundle.
+
+Once the machines are provisioned, you will see a table of their current statuses as the software is automatically configured.
+
+![IMAGE](https://assets.ubuntu.com/v1/1e9223dc-machine-status.png)
+
+## Additional configuration
+Duration: 3
+
+You will be asked if you want to download and install [kubectl](https://kubernetes.io/docs/user-guide/kubectl-overview/) and [kubefed](https://kubernetes.io/docs/admin/kubefed/) on your local machine. These are command line utilities for interacting with your new Kubernetes cluster.
+
+Check the box to install the applications and enter your sudo password when prompted.
+
+![image](https://assets.ubuntu.com/v1/430ae051-sudo-install.png)
+
+## That’s all folks!
+
+Congratulations, you have installed and configured a cluster running the Canonical Distribution of Kubernetes.
+
+### Next steps
+
+Now that you have your cluster, you can put it to work:
+
+* [The easy way to commoditise GPUs for Kubernetes][kubegpu]
+* [Build a transcoding platform in minutes][kubetransform]
+* [Transform your solution into a private PaaS][kubepaas]
+
+### Further reading
+
+* Learn more about the [Canonical Distribution of Kubernetes][canonicalkubernetes] bundle
+* Discover [Kubernetes][cankube]
+* Get involved and connect with the [Kubernetes community][kubecommunity]
+
+<!-- LINKS -->
```

Comment:
You have a lot of links that are not used here. Could you clean it up or add the links where they need to be?

Reply:
Oh, yeah. I had intended to, good catch. I coped this from the other K8s tutorial but it needs to be cleaned.